eng/Tools.props

```diff
@@ -45,8 +45,7 @@
     <PackageReference Include="System.Security.Cryptography.Hashing.Algorithms.Analyzers" Version="1.1.0" />
 
     <!-- Include the Roslyn Compilers -->
-    <PackageReference Include="Microsoft.Net.Compilers" Version="$(MicrosoftNetCompilersVersion)" PrivateAssets="all" ExcludeAssets="Build" GeneratePathProperty="true" IsImplicitlyDefined="true" />
-    <PackageReference Include="Microsoft.NETCore.Compilers" Version="$(MicrosoftNETCoreCompilersVersion)" PrivateAssets="all" ExcludeAssets="Build" GeneratePathProperty="true" IsImplicitlyDefined="true" />
+    <PackageReference Include="Microsoft.Net.Compilers.Toolset" Version="$(MicrosoftNetCompilersToolsetVersion)" PrivateAssets="all" IsImplicitlyDefined="true" />
```

Comment:
Seems we still need this and the package references aren't being picked up from Arcade automatically.

@ericstj, @safern is this expected?

Reply:
(Versions and everything else appear to be, just not some of the package references)

Reply:
If we want to update version of this package automatically we need to add it into Version.Details.xml and then add a subscription on BAR to roslyn. Which we currently don't have

Reply:
CoreFX has very customized nuget restore.

Reply:
Yeah. We took this thread offline now. We where wondering if it would be correct to import the sdk.props in our Tools.props so that we get this opt-in package references by arcade directly rather than having to duplicate them in our Tools.props

Reply:
No, that would not be correct. CoreFX opted into this model and specifically did not want to get any implicit packages.